.gitignore

```diff
@@ -102,4 +102,7 @@ dist
 .dynamodb/
 
 # TernJS port file
-.tern-port
\ No newline at end of file
+.tern-port
+
+.vscode/
```

Comment:
We shouldn't gitignore the `.vscode` since this contains settings and commands that are important for the application to run (e.g. Azure Functions settings). They don't (or shouldn't) include local user settings so it's advisable to have this in source control.